container-images/scripts/build_llama_and_whisper.sh

```diff
@@ -133,7 +146,29 @@ clone_and_build_ramalama() {
   rm -rf ramalama
 }
 
+clean_fedora_rocm() {
+  case "${1}" in 
+    gfx8)
+      rm -fr /usr/lib64/rocm/gfx9* /usr/lib64/rocm/gfx10* /usr/lib64/rocm/gfx11*
+      ;;
+    gfx9)
+      rm -fr /usr/lib64/rocm/gfx8* /usr/lib64/rocm/gfx10* /usr/lib64/rocm/gfx11* 
+      ;;
+    gfx10)
+      rm -fr /usr/lib64/rocm/gfx8* /usr/lib64/rocm/gfx9* /usr/lib64/rocm/gfx11* 
+      ;;
+    gfx11)
+      rm -fr /usr/lib64/rocm/gfx8* /usr/lib64/rocm/gfx9* /usr/lib64/rocm/gfx10* && \
+      ln -s /usr/lib64/rocm/gfx1103/lib/rocblas/library/TensileLibrary_lazy_gfx1103.dat \
```

Comment:
Interesting you figured out we need to do this, worth alerting Tom Rix to this, it might be better to fix this in the rpm package. Worth checking if this is needed for UBI9 also

Reply:
@ericcurtin that isn't needed in the Rawhide and Fedora 42 nightlies of ROCm but support for gfx1103 is considered experimental in Fedora 41 so I'm not sure where the fix is appropriate.

Reply:
Yes.  gfx1103 was experimental in F41.
gfx1035,gfx1103,gfx1150,gfx1151,gfx1152,gfx1200,gfx1201 were added in F42.